### ISSUE TITLE
add a client name to /node/version

### ISSUE THREAD

**Issue by tzapu** [prysmaticlabs/prysm#4238]
it would be useful to have a client name on https://api.prylabs.net/eth/v1alpha1/node/version
so it would look something like :
```
{
 "version": "Prysm/Git commit: 7ccbe48f54b50bae541307d61fd5b459e4faa314. Built at: 2019-12-09 14:49:56+00:00",
 "metadata": ""
}
```
maybe. or any other combination you think is appropriate.
Cheers

**Comment by prestonvanloon** [comment]
I agree. Thanks for the suggestion!

**Comment by metanull-operator** [comment]
I'm happy to pick this one up, if you want to assign it to me. Do you want to go with the suggestion, or is there something else you'd like to have in the version information? Any other changes, other than adding "Prysm"?

**Comment by terencechain** [comment]
The suggestion is good with me

Minor modifications can happen in the PR

**Comment by metanull-operator** [comment]
This can be closed. Are there any steps I can take for that?

**Comment by rauljordan** [comment]
Hey @metanull-operator typically if you put the words `Resolves #4238` in your PR corresponding to this issue, once the PR is merged into master, the issue will automatically close

**Comment by metanull-operator** [comment]
I suppose the square brackets in the PR template are supposed to be removed? I had "[Resolves]" in my PR. That must explain it.
